### PR TITLE
MCKIN-17533 Korean translations added v3

### PR DESCRIPTION
**Pull request by musmanmalik**
Korean translations added v3

### CODE REVIEW

**Comment by musmanmalik** [comment]
@xitij2000 Please review these translation PRs. Thanks

File: problem_builder/public/js/translations/ko_KR/textjs.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+}(this));
+
+
+                  }
+                };
+                ProblemBuilderXBlockI18N.init();
+                global.ProblemBuilderXBlockI18N = ProblemBuilderXBlockI18N;
+            }(this));
+
```
[the remark this answers is not in the document]
Comment (musmanmalik) [reply problem_builder/public/js/translations/ko_KR/textjs.js:165]: @taniwha Answer is given by @xitij2000 on https://github.com/open-craft/problem-builder/pull/283.

[the remark this answers is not in the document]
Comment (taniwha) [reply problem_builder/public/js/translations/ko_KR/textjs.js:165]: as per #283, accepting

**Comment by musmanmalik** [comment]
@taniwha Thanks for merging this one more step needed. Can you please create release tag for v3?

**Comment by taniwha** [comment]
Ah, yeah, I'll look into what I need to do right now

**Comment by musmanmalik** [comment]
@xitij2000 @taniwha  Its just an observation newly created tags are looks different than previous one. Is this fine?
Screenshot 2020-06-04 at 12 35 26 PM

**Comment by taniwha** [comment]
Yes, that should be fine, especially if you use the tag for checkout. It's because I created the releases via command-line (`git tag`) rather than github due to  double checking the version numbers.
I'll double-check, though.

**Comment by taniwha** [comment]
@musmanmalik while I expect it was not an issue, I have sorted out the differences in order to cause less confusion. My apologies.

**Comment by musmanmalik** [comment]
Thank you so much @taniwha .

**Pull request merged**